spec/models/user_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  describe "validations" do
+    it {should validate_presence_of(:email)}
+    it {should validate_uniqueness_of(:email)}
+    it {should validate_presence_of(:password)}
+  end
+
+  describe "relationships" do
+  end
```

Comment:
This is where we will begin as we do the self-referential joins for users and their "friends"

Reply:
NOICE!